lib/ui/views/installer/installer_viewmodel.dart

```diff
@@ -338,10 +415,6 @@ class InstallerViewModel extends BaseViewModel {
     }
   }
 
-  void exportLog() {
-    _patcherAPI.exportPatcherLog(logs);
-  }
-
```

Comment:
Considering this makes exportPatcherLog unused, maybe it should be deleted?

Reply:
This was marked as resolved, but I don't see any changes about this

Reply:
You're suggesting that `exportLog` be removed right? Well it has been removed, even before you requested the change.

Reply:
Oh wait... I'm dumb. It was `_patcherAPI.exportPatcherLog` you were wanting removed. My bad

Reply:
Sorry about that.

Reply:
I actually think it should be left in, because there was some talk about maybe in the future that exporting to file might return if it is required.

Reply:
I see, fine by me